Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Retrieve_File_README.md

```diff
@@ -0,0 +1,32 @@
+This playbook uses the Live Response feature in order to retrieve a file from an endpoint.
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+This playbook does not use any sub-playbooks.
+
+### Integrations
+* MicrosoftDefenderAdvancedThreatProtection
+
+### Scripts
+* isError
+* UnzipFile
+
+### Commands
+* microsoft-atp-live-response-get-file
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
```

Comment:
No input for host ID?

Reply:
I didn't see an input for host ID in the yaml

Reply:
@David-BMS - No, this PB was intended to retrieve multiple files from multiple endpoints by using looping only the file paths.  Did you change the logic on your end?

Reply:
Spoke with @David-BMS , This PB is looping on File Paths and the Endpoint id will be retrieved from the incident field 'Device ID'. 
@julieschwartz18 - I have changed the description a bit - so the users will be aware of this.
`"This playbook uses the Live Response feature to retrieve a file from an endpoint./nNote that the endpoint id will be set from the incident field \"Device ID\"."`